rs/bitcoin/kyt/tests/tests.rs

```diff
@@ -0,0 +1,115 @@
+use candid::{Decode, Encode, Principal};
+use ic_test_utilities_load_wasm::load_wasm;
+use pocket_ic::{
+    common::rest::{
+        CanisterHttpReply, CanisterHttpRequest, CanisterHttpResponse, MockCanisterHttpResponse,
+    },
+    PocketIc, WasmResult,
+};
+
+const MAX_TICKS: usize = 10;
+
+fn kyt_wasm() -> Vec<u8> {
+    load_wasm(
+        std::env::var("CARGO_MANIFEST_DIR").unwrap(),
+        "ic-btc-kyt-canister",
+        &[],
+    )
+}
+
+#[test]
+fn test_get_inputs() {
+    let env = PocketIc::new();
+    let sender = Principal::anonymous();
+
+    let kyt = env.create_canister();
+    env.add_cycles(kyt, 100_000_000_000_000);
+    env.install_canister(kyt, kyt_wasm(), vec![], None);
+
+    let call_id = env
+        .submit_call(
+            kyt,
+            sender,
+            "get_inputs",
+            Encode!(
+                &"c80763842edc9a697a2114517cf0c138c5403a761ef63cfad1fa6993fa3475ed".to_string()
+            )
+            .unwrap(),
+        )
+        .expect("submit_call failed to return call id");
+
+    let canister_http_requests = tick_until_next_request(&env);
+    env.mock_canister_http_response(MockCanisterHttpResponse {
+        subnet_id: canister_http_requests[0].subnet_id,
+        request_id: canister_http_requests[0].request_id,
+        response: CanisterHttpResponse::CanisterHttpReply(CanisterHttpReply {
+            status: 200,
+            headers: vec![],
+            body: b"\
+\x02\x00\x00\x00\x01\x17\x34\x3a\xab\xa9\x67\x67\x2f\x17\xef\x0a\xbf\x4b\xb1\x14\xad\x19\x63\xe0\
```

Comment:
nit: Could you maybe add a comment explaining how those responses were obtained?

Reply:
Good idea! Add a comment in [9fcc9f4](https://github.com/dfinity/ic/pull/1150/commits/9fcc9f4600d5b787be55b429033963c7808ff3fe)

Reply:
Thanks, that's exactly what I was looking for!